src/main/java/amazonreviewsapp/springboot/controller/ReviewController.java

```diff
@@ -104,6 +108,7 @@ public boolean deleteReviewById(@PathVariable String id) {
     }
 
     private User getUserFromRequest(HttpServletRequest request) {
-        return new User();
+        String token = provider.resolveToken(request);
```

Comment:
Do you need to use provider in each controller since you have added filter?

Reply:
My jwt filter checks if the token is present and valid, but this method helps me to get User by this token (I mean the current authenticated User object)